Plugins/Public/munitioncntl/Main.cpp

```diff
@@ -171,7 +171,8 @@ void ProcessGuided(FLPACKET_CREATEGUIDED& createGuidedPacket)
 	{
 		case NOTRACK_NOALERT:
 		{
-			const auto& projectile = reinterpret_cast<CGuided*>(CObject::Find(createGuidedPacket.iProjectileId, CObject::CGUIDED_OBJECT));
+			CGuided* projectile = reinterpret_cast<CGuided*>(CObject::Find(createGuidedPacket.iProjectileId, CObject::CGUIDED_OBJECT));
+			projectile->Release();
```

Comment:
Shouldn't this be after we call set_target to ensure we're not about to try to interact with free'd memory?